UI/NativeApp.cpp

```diff
@@ -498,10 +498,10 @@ void NativeInit(int argc, const char *argv[], const char *savegame_dir, const ch
 	else if (getenv("HOME") != NULL)
 		config = getenv("HOME") + std::string("/.config");
 	else // Just in case
-		config = "./config";
+		config = "config";
 
-	g_Config.memStickDirectory = config + "/ppsspp/";
-	g_Config.flash0Directory = File::GetExeDirectory() + "/assets/flash0/";
+	g_Config.memStickDirectory = g_Config.internalDataDirectory + config + "/ppsspp/";
+	g_Config.flash0Directory = g_Config.internalDataDirectory + "assets/flash0/";
```

Comment:
These changes shouldn't be needed (or the above), since it should be using the `#elif PPSSPP_PLATFORM(SWITCH)` path on switch.

-[Unknown]

Reply:
Ok I can double check this.